cocos/ui/UIEditBox/iOS/CCUISingleLineTextField.mm

```diff
@@ -45,4 +41,47 @@ - (CGRect)editingRectForBounds:(CGRect)bounds
     return [self textRectForBounds:bounds];
 }
 
+- (void)setInnerPadding:(float)left :(float)top :(float)right :(float)bottom
+{
+    auto glview = cocos2d::Director::getInstance()->getOpenGLView();
+    float factor = glview->getScaleX() / glview->getContentScaleFactor();
+    
+    _leftPadding = left * factor;
+    _topPadding = top * factor;
+    _rightPadding = right * factor;
+    _bottomPadding = bottom * factor;
+    
+}
+
+- (void)setTextAlignment:(cocos2d::TextHAlignment)hAlign :(cocos2d::TextVAlignment)vAlign
+{
+    switch (hAlign) {
+        case cocos2d::TextHAlignment::LEFT :
+            [self setTextAlignment:NSTextAlignmentLeft];
```

Comment:
@Mazyod  In cocos2d-x 3.9 I see deployment target is 6.0.

https://github.com/cocos2d/cocos2d-x/commit/4633a9363168bd4647932263b85ee799fad7f083